app/concerns/web_request_concern.rb

```diff
@@ -2,6 +2,18 @@
 require 'faraday_middleware'
 
 module WebRequestConcern
+  module DoNotEncoder
+    def self.encode(params)
+      params.map do |key, value|
+        value.present? ? "#{key}=#{value}" : "#{key}"
```

Comment:
Shouldn't it be `value.nil? ? "#{key}" ? "#{key}=#{value}"`, considering `"".present?` is false?